js/utils.js

```diff
@@ -557,12 +559,12 @@ const dashBreak = '<p>-----------------------------------</p>';
  * - window.OressourceEnv.types_evac
  * - window.OressourceEnv.types_dechet
  */
-function showTickets(data, unit = 'kg') {
+function showTickets(data) {
   const item = (data.hasOwnProperty('items') && data.items.length > 0
-    ? `<p>Ventes</p>${dashBreak}${showTicket(data.items, window.OressourceEnv.types_dechet, unit)}`
+    ? `<p>Ventes</p>${dashBreak}${showTicket(data.items, window.OressourceEnv.types_dechet)}`
```

Comment:
`unit` was passed around and in `showTicket`[link to fct](https://github.com/mart1ver/oressource/pull/413/files#diff-1e20a0ee0fa5c5689cd8541a5892e5e23248b3bf2ab918043e525c443856f18cR578-R590) it wasn't read at all.